scripts/2022_07_11_update_minichef_rate_arbitrum.py

```diff
@@ -0,0 +1,60 @@
+from helpers import CHAIN_IDS, MINICHEF_ADDRESSES, MULTISIG_ADDRESSES
+from ape_safe import ApeSafe
+from brownie import ZERO_ADDRESS, Contract, accounts, network
+
+from scripts.utils import confirm_posting_transaction, fetch_current_nonce
+
+
+def main():
+    """
+    Updates Arbitrum pool and minichef parameters
+    Existing pools' admin fees are updated to 50% of the total swap fee
+    Minichef rate and allocpoint are updated to match Snapshot results
+    """
+
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")  # prompts for password
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["ARBITRUM"]])
+
+    minichef = multisig.contract(MINICHEF_ADDRESSES[CHAIN_IDS["ARBITRUM"]])
+
+    pool_addrs = [
+        "0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9",  # arbUSDPool
+        "0xfeEa4D1BacB0519E8f952460A70719944fe56Ee0",  # arbUSDPoolv2
+        "0x5dD186f8809147F96D3ffC4508F3C82694E58c9c",  # usdsMetaPool
+    ]
+
+    for pool_addr in pool_addrs:
+        pool = multisig.contract(pool_addr)
+        pool.setAdminFee("5000000000")  # .5 x 10**10, set to 50% of swap fee
```

Comment:
In case you weren't aware and you're using python version 3.6+ you can use _ to group numbers by the thousands to help readability.